app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -84,14 +84,13 @@ class BloodPressureHistoryScreen(
 
   private val delegate: MobiusDelegate<BloodPressureHistoryScreenModel, BloodPressureHistoryScreenEvent, BloodPressureHistoryScreenEffect> by unsafeLazy {
     val screenKey = screenRouter.key<BloodPressureHistoryScreenKey>(this)
-    MobiusDelegate(
+    MobiusDelegate.forView(
         events = events,
         defaultModel = BloodPressureHistoryScreenModel.create(screenKey.patientUuid),
         init = BloodPressureHistoryScreenInit(),
         update = BloodPressureHistoryScreenUpdate(),
         effectHandler = effectHandler.create(this).build(),
-        modelUpdateListener = uiRenderer::render,
-        crashReporter = crashReporter
```

Comment:
I think the `crashReporter` property is no longer used.

Reply:
Yes, will remove the inject 👍